docs/examples.md

```diff
@@ -372,6 +390,23 @@ systemd:
         [Install]
         WantedBy=multi-user.target
 ```
+
+This example specifies a systemd unit (`example.service`) and a dropin (`proxy.conf`) to be read from local files at transpile time. The file paths are relative to the directory specified with the `--files-dir` command-line option, which must be provided.
+
+<!-- butane-config -->
+```yaml
+variant: fcos
+version: 1.5.0
+systemd:
+  units:
+    - name: example.service
+      contents_local: example.service
+    - name: rpm-ostreed.service
+      dropins:
+          - name: proxy.conf
```

Comment:
Maybe it'd be clearer to use `example.conf` here (and in the description paragraph) rather than splitting the difference on naming?

Reply:
Personally, I like the approach of having a file reference differently named from the target name of the unit we modify because it shows multiple things at once:

- how to modify a unit via dropins
- how file referencing works to provide unit/config contents
- that the file reference itself (its path/name) has _nothing_ to do with the actual unit that will be put into the Ignition config, i.e. it's up to the user how to arrange and name their local files

I like slightly more verbose examples that do not easily allow one to jump to conclusions such as "oh they are called the same so … I guess that's what I have to do to make it work". If you think this verbose approach to be unwarranted (in this example) I'll remove it, no worries.

Reply:
Yeah, that makes sense.  I was partially reacting to the inconsistency: the dropin file uses a different name but the service doesn't.  Was that an oversight, or are you specifically looking to show that the local filename may or may not match the target filename?

Reply:
Effectively the latter I suppose, i.e. the local file name may or may not match, however, with a single example this is difficult to show (as our discussion shows) because of perceived inconsistencies in the example itself. I have no really strong feelings one way or another and I think that users will figure it out correctly anyway. It's not rocket science after all :)